packages/terra-tabs/src/common-tabs/_HiddenTab.jsx

```diff
@@ -148,6 +165,36 @@ const HiddenTab = ({
   attributes['data-focus-styles-enabled'] = true;
   attributes['aria-selected'] = isSelected;
 
+  const onFocusResponse = intl.formatMessage({ id: 'Terra.tabs.focus' });
+  const responseId = `terra-hidden-tab-pane-response=${uuidv4()}`;
+
+  if (isDraggable) {
+    return (
+      <Draggable key={id} draggableId={id} index={index}>
+        {(provided) => (
+          <div
+            {...attributes}
+            ref={provided.innerRef}
+            {...provided.draggableProps}
+            {...provided.dragHandleProps}
+            id={id}
+            aria-controls={associatedPanelId}
+            role="tab"
+            className={hiddenClassNames}
+            aria-disabled={isDisabled}
+            aria-describedby={responseId}
+          >
+            <VisuallyHiddenText aria-hidden id={responseId} text={onFocusResponse} />
+            <IconKnurling />
+            <div className={cx('checkbox')}>{isSelected ? <IconCheckmark /> : null}</div>
+            {showIcon && <div>{icon}</div>}
+            <div className={cx('label', { 'with-icon': showIcon })}>{label}</div>
+          </div>
+        )}
+      </Draggable>
```

Comment:
Are we ensuring that Add button which gets added as part of different PR will be out of draggable context.

Reply:
Can be achieved by adding `isDragDisabled={condition which is true for add tab only}`. Will add it in the PR which is merged later.